.github/workflows/build-container.yml

```
@@ -0,0 +1,35 @@
+name: build-docker-image
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '7 0 * * sun'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+
+    steps:
+
+    - name: checkout docker-software-base
+      uses: actions/checkout@v2
+
+    - name: make date tag
+      id: mkdatetag
+      run: date +%Y%m%d-%H%M
+
+    - name: Build and push Docker images
+      uses: docker/build-push-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        repository: opensciencegrid/software-base
+        tags:
+          - fresh
+          - ${{ steps.mkdatetag.outputs.dtag }}
```

```suggestion
        tags: fresh, ${{ steps.mkdatetag.outputs.dtag }}
```

This needs to be comma-separated:https://github.com/docker/build-push-action#tags

---

Ah, i misunderstood, thinking it was supposed to be a yaml list.

The [example](https://github.com/docker/build-push-action#tags) shows `tags: tag1,tag2` without any whitespace.  Is whitespace optional?  If not it looks like there's an extra space in your suggested change.

---

```suggestion
        tags: fresh,${{ steps.mkdatetag.outputs.dtag }}
```

Beats me! Here's a suggestion without the whitespace